internal/redaction/sink_policy_test.go

```
@@ -0,0 +1,162 @@
+	yes := true
+	if got := SinkPolicyForDirective(&yes); got != SinkPolicyRedact {
+		t.Fatalf("true directive = %v, want SinkPolicyRedact", got)
+	}
+	no := false
+	if got := SinkPolicyForDirective(&no); got != SinkPolicyRaw {
+		t.Fatalf("false directive = %v, want SinkPolicyRaw", got)
+	}
+}
+
+func TestSinkPolicyContextRoundTrip(t *testing.T) {
+	if got := SinkPolicyFromContext(nil); got != SinkPolicyDefault {
```

golangci-lint (2.12.2)

[error] SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use

(staticcheck)